.editorconfig

```diff
@@ -0,0 +1,12 @@
+# http://editorconfig.org
+root = true
+
+[*]
+indent_style = space
+indent_size = 2
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+
+[*.md]
+trim_trailing_whitespace = false
```

Comment:
This file is not related to the suffix option. I'd prefer to get this changes merged in a separate PR, so that we can also look for other common editor and linter configurations.